README.md

```diff
@@ -19,6 +19,15 @@ The three co-leads on this project are @colinxfleming (rails and technical lead)
 
 ## Contributing to this Project
 
+<<<<<<< HEAD
```

Comment:
Hey @mvaden , we took this part out of the readme since it's no longer hacktoberfest. Can you do me a solid and take out line 22 thru 30?